GliaWidgets/Sources/EntryWidget/EntryWidget.swift

```diff
@@ -0,0 +1,162 @@
+import Foundation
+import UIKit
+import SwiftUI
+
+public final class EntryWidget: NSObject {
```

Comment:
Does it need to be `NSObject`?

Reply:
Unfortunately yes, it is required for `UIViewControllerTransitioningDelegate`

Reply:
@EgorovEI probably yes, since this class is also delegate, line 104: ` hostingController.transitioningDelegate = self`

Reply:
I missed it is delegate